### PR TITLE
DAT-2156 Curated content translations

### PR DESCRIPTION
**Pull request by pawelchcki**

### CODE REVIEW

**Comment by pchojnacki** [comment]
see #5948

Tests are failing due to unrelated code that is not pushed in this change.

**Comment by pchojnacki** [comment]
@sqreek CR

File: extensions/wikia/CuratedContent/CuratedContent.i18n.php [hunk 1]
```diff
@@ -61,116 +44,190 @@
...
+	'wikiacuratedcontent-content-description-section' => 'crwdns6985:0crwdne6985:0',
+	'wikiacuratedcontent-content-description-organize' => 'crwdns6986:0crwdne6986:0',
+	'wikiacuratedcontent-content-description-no-section' => 'crwdns6987:0crwdne6987:0',
+	'wikiacuratedcontent-content-description-items-input' => 'crwdns6988:0{ns:Category}crwdnd6988:0{ns:500}crwdnd6988:0{ns:File}crwdne6988:0',
+	'wikiacuratedcontent-content-articlenotfound-error' => 'crwdns6989:0crwdne6989:0',
+	'wikiacuratedcontent-content-emptylabel-error' => 'crwdns7627:0crwdne7627:0',
+	'wikiacuratedcontent-content-videonotsupported-error' => 'crwdns6991:0crwdne6991:0',
+	'wikiacuratedcontent-content-notsupportedtype-error' => 'crwdns6992:0crwdne6992:0',
+	'wikiacuratedcontent-content-nocategoryintag-error' => 'crwdns6993:0crwdne6993:0',
+	'wikiacuratedcontent-featured-section-name' => 'crwdns6994:0crwdne6994:0',
```
Comment (sqreek) [inline extensions/wikia/CuratedContent/CuratedContent.i18n.php]: this section look strange.

**Comment by sqreek** [comment]
except acholi dialect lgtm

File: extensions/wikia/CuratedContent/CuratedContent.i18n.php [hunk 2]
```diff
@@ -61,116 +44,190 @@
...
-);
-
-/** Asturian (asturianu)
- * @author Xuacu
- */
-$messages[ 'ast' ] = array(
+	'wikiacuratedcontent-featured-section-name' => 'Featured content section title',
+);
+
+$messages['ach'] = array(
```
Comment (pchojnacki) [inline extensions/wikia/CuratedContent/CuratedContent.i18n.php]: @lgarczewski @Grunny 
ach - language ? should I leave it here ?

**Pull request merged**